internal/testsuite/app.go

```diff
@@ -232,6 +234,21 @@ func (a *App) Test(ctx context.Context, testSpec *api.TestSpec, asserters ...fun
 	watcher.Out = a.Out
 	g.Go(func() error { return watcher.Run(ctx) })
 
+	// TODO: handle better
+	kubectxs := []string{
+		"arn:aws:eks:us-east-1:235758441054:cluster/dev-armada",
```

Comment:
Are there any security issue exposing these in a public repo?

Reply:
I added a comment on this block, this is WIP, I am still not sure how and where will we run testsuite, will it be in-cluster or out-of-cluster, this definitely will be removed in future commits to this PR